cmd/crane/cmd/export.go

```diff
@@ -45,9 +51,20 @@ func NewCmdExport(options *[]crane.Option) *cobra.Command {
 			}
 			defer f.Close()
 
-			img, err := crane.Pull(src, *options...)
-			if err != nil {
-				return fmt.Errorf("pulling %s: %w", src, err)
+			var img v1.Image
+			if src == "-" {
+				img, err = tarball.Image(func() (io.ReadCloser, error) {
+					return io.ReadCloser(os.Stdin), nil
+				}, nil)
+				//img, err = tarball.ImageFromPath("test.tar", nil)
```

Comment:
Drop this?

Reply:
Just a reminder that there is an alternative API that may work if the file is first saved to disk from `os.Stdin`. I will move it to the issue comment instead.

Reply:
Dropped.